Real_Masters_all/burtonml.xml

```diff
@@ -133,7 +133,7 @@
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no98115213">Babst, Earl D., 1870-1967.</persname>
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/n50017666">Balfour, Arthur James, 1848-1930.</persname>
         <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2001090656">Bates, Henry Moore, 1869-1949.</persname>
-        <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2008186663">Beal, Junius E. (Junius Emery), 1860-1942.</persname>
+        <persname source="lcnaf" encodinganalog="700" authfilenumber="http://id.loc.gov/authorities/names/no2008186663">Beal, Junius Emery, 1860-1942.</persname>
```

Comment:
Hi Walker,

I apologize, I should have given you some more context. In this case, you've picked the older format. Here's the new format:

http://lccn.loc.gov/n94104489
Authorized (i.e., we want this one): Damon, Robert H. (Robert Harris), 1908-
Variant (i.e., the old way): Nutt, Damon Harris, 1908-

Yeah, sorry about that. Dallas and I are in a meeting now but we can talk about this after.